python-sdk/nuscenes/eval/detection/render.py

```diff
@@ -95,7 +101,7 @@ def visualize_sample(nusc: NuScenes,
     # Show / save plot.
     if verbose:
         print('Rendering sample token %s' % sample_token)
-    plt.title(sample_token)
+        plt.title(sample_token)
```

Comment:
Why is this change needed?

Reply:
Users  uses verbose to determine whether to display the title

Reply:
`verbose` should only be used to control `print`s - let's undo this change

Reply:
ok, and it has done